.env.example

```diff
@@ -8,18 +8,17 @@ ENCRYPTION_KEY=6c1fe4e407b8911c104518103505b218
 # THIS IS A SAMPLE AUTH_SECRET KEY AND SHOULD NEVER BE USED FOR PRODUCTION
 AUTH_SECRET=5lrMXKKWCVocS/uerPsl7V+TX/aaUaI7iDkgl3tSmLE=
 
-# MongoDB
-# Backend will connect to the MongoDB instance at connection string MONGO_URL which can either be a ref
-# to the MongoDB container instance or Mongo Cloud
+# Postgres URL 
+# Backend will connect to the Postgres instance at connection string DB_CONNECTION_URI which can either be a ref
 # Required
-MONGO_URL=mongodb://root:example@mongo:27017/?authSource=admin
+DB_CONNECTION_URI=postgres://infisical:infisical@db:5432/infisical
 
 # Redis
 REDIS_URL=redis://redis:6379
 
-# Optional credentials for MongoDB container instance and Mongo-Express
-MONGO_USERNAME=root
-MONGO_PASSWORD=example
+POSTGRES_PASSWORD=infisical
+POSTGRES_USER=infisical
+POSTGRES_DB=infisical
```

Comment:
Versioning is a great idea @Grraahaam

Reply:
@Grraahaam Good suggestion but we do hope this would be our last database change. We will be only supporting postgres it won't be other SQL because we are using some exclusive  PG things. Also we don't wanna handler other SQL drivers bug too